frontend/src/app/corpus-info/corpus-info.component.ts

```diff
@@ -41,4 +46,12 @@ export class CorpusInfoComponent implements OnInit {
         return marked.parse(content);
     }
 
+    private sortPages(pages: CorpusDocumentationPage[]): CorpusDocumentationPage[] {
+        return _.sortBy(pages, 'index');
+    }
+
+    private includePage(corpus: Corpus, page: CorpusDocumentationPage): boolean {
+        return page.type !== 'Word models' || corpus.wordModelsPresent;
```

Comment:
Add some parentheses to this expression to ensure to everyone that it does what is expected.